libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/ExecSpecRule/Conway/Certs.hs

```diff
@@ -74,7 +69,7 @@ instance
               credsSet = Set.fromList creds
               zeroRewards (Agda.MkHSMap pairs) =
                 Agda.MkHSMap (map (\(c, r) -> if c `Set.member` credsSet then (c, 0) else (c, r)) pairs)
-      _ -> checkConformance @"CERTS" @Conway @fn ctx env st sig implResTest agdaResTest
+      _ -> checkConformance @"CERTS" @Conway @fn ctx env st sig (first undefined implResTest) agdaResTest
```

Comment:
What is the intention of this change? 
So here we are in the case in which `(implResTest, agdaResTest)` are not `(Right _, Right _)`. 
Is it like this:  run `checkConformance` only in case `implResTest` is a `Right`, otherwise fail with undefined? 
I realize I don't actually understand  this branch even before the change - if both are not Right, we run the same without fixup? How come?